src/SimplifiedModelControllers/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+if(FRAMEWORK_COMPILE_SimplifiedModelControllers)
+
+  set(H_PREFIX include/BipedalLocomotion/SimplifiedModelControllers)
+
+  add_bipedal_locomotion_library(
+    NAME                   SimplifiedModelControllers
+    PUBLIC_HEADERS         ${H_PREFIX}/CoMZMPController.h
+    SOURCES                src/CoMZMPController.cpp
+    PUBLIC_LINK_LIBRARIES  Eigen3::Eigen BipedalLocomotion::ParametersHandler MANIF::manif
```

Comment:
After looking at the CI failure, I guess that a link to the ``System`` component is missing

Reply:
Done https://github.com/dic-iit/bipedal-locomotion-framework/pull/387/commits/1b8765e55088745ccbbe0fbddbd3ef227dc87eb9